packages/core/data-transfer/src/strapi/providers/remote-destination/index.ts

```diff
@@ -78,33 +101,48 @@ class RemoteStrapiDestinationProvider implements IDestinationProvider {
       return new ProviderTransferError('Unexpected error');
     }
 
+    this.stats[step] = { count: 0 };
+
     return null;
   }
 
   async #endStep<T extends Client.TransferPushStep>(step: T) {
     try {
-      await this.dispatcher?.dispatchTransferStep({ action: 'end', step });
+      const res = await this.dispatcher?.dispatchTransferStep<{
+        ok: boolean;
```

Comment:
I think we even have a type for the return value of those dispatch calls (in the protocol types). Could you just verify we can't re-use/modify them instead of writing it raw here